templates/base/graphQl/ApolloClient.js

```diff
@@ -0,0 +1,21 @@
+import ApolloClient, { HttpLink, InMemoryCache } from 'apollo-boost';
+
+const defaultUri = 'https://48p1r2roz4.sse.codesandbox.io';
```

Comment:
Should be coming from an `templates/base/config/api/index.js ` file

Reply:
done